robots/accession/remediate_object.rb

```diff
@@ -11,7 +11,7 @@ def initialize
 
         def perform(druid)
           obj = Dor.find(druid)
-          obj.upgrade!
+          obj.upgrade! if obj.respond_to? :upgrade!
```

Comment:
yes, indeed that functionality was removed...but there was some discussion with Lynn that it in the short to medium term was better to continue making the call if it exists just in case we wanted to re-implement some part of "upgrade!" later (robot becomes a noop now, but may resume doing something later)

if that doesn't happen in some intermediate term, we would remove the robot entirely